src/main/java/bio/terra/stairway/FlightDao.java

```diff
@@ -56,6 +56,7 @@
   static final String FLIGHT_TABLE = "flight";
   static final String FLIGHT_LOG_TABLE = "flightlog";
   static final String FLIGHT_INPUT_TABLE = "flightinput";
+  static final int FLIGHT_PARAMETERS_VERSION = 2;
```

Comment:
nit: maybe `CURRENT_PARAMETERS_VERSION`?

Reply:
Thinking about this, I think "parameters" bound this too much, and I think this should probably change to something like `DATABASE_SCHEMA_VERSION`, so that we have one constant to bump on schema updates.